### ISSUE TITLE
0.31.x: ramses_tx.set_pkt_logging_config() argument after ** must be a mapping, not str

### ISSUE THREAD

**Issue by zxdavb** [zxdavb/evohome#96]
```
> python client.py -rrr listen /dev/ttyACM0 -o packet.log
```
... gives:
```
client.py: Starting ramses_rf...
 - sending is force-disabled
22:10:28.726 Best practice is exactly one gateway (HGI) in the known_list: []
22:10:28.726 It is strongly recommended to provide a known_list, and use it as a whitelist (device_id filter), configure: enforce_known_list = True
Traceback (most recent call last):
  File "/home/dbonnes/clients/ramses_rf/client.py", line 19, in <module>
    main()
  File "/home/dbonnes/clients/ramses_rf/src/ramses_cli/client.py", line 595, in main
    asyncio.run(async_main(command, lib_kwargs, **kwargs))
  File "/usr/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/dbonnes/clients/ramses_rf/src/ramses_cli/client.py", line 507, in async_main
    gwy = Gateway(serial_port, **lib_kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dbonnes/clients/ramses_rf/src/ramses_rf/gateway.py", line 129, in __init__
    set_pkt_logging_config(
TypeError: ramses_tx.set_pkt_logging_config() argument after ** must be a mapping, not str
```

**Comment by renini** [comment]
Have the same thing, although `python3 client.py monitor /dev/ttyACM0` without -o seems to work.
In my case python 3.11.2 by the way.

**Comment by zxdavb** [comment]
Fixed by commit 57d255516dbb2393b8e38cb8944a4487827aee28
